apps/backoffice-v2/src/pages/MerchantMonitoringBusinessReport/hooks/useMerchantMonitoringBusinessReportLogic/useMerchantMonitoringBusinessReportLogic.tsx

```
@@ -1,23 +1,27 @@
+import { ParsedBooleanSchema } from '@ballerine/ui';
 import { t } from 'i18next';
 import { toast } from 'sonner';
 import { capitalize } from 'lodash-es';
-import { isObject } from '@ballerine/common';
-import { useCallback, useMemo } from 'react';
-import { zodResolver } from '@hookform/resolvers/zod';
+import { useCallback } from 'react';
 import { SubmitHandler, useForm } from 'react-hook-form';
 import { useNavigate, useParams } from 'react-router-dom';
 import { useLocale } from '@/common/hooks/useLocale/useLocale';
 import { ParsedBooleanSchema, useReportTabs } from '@ballerine/ui';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'ParsedBooleanSchema'. Consider to delete it or rename it.

'ParsedBooleanSchema' is defined here:


(lint/suspicious/noRedeclare)